### ISSUE TITLE
set (default) rowheight

### ISSUE THREAD

**Issue by thomrob** [BBj-Plugins/BBjGridExWidget#110]
Looking for a way to change the rowheight, and i'm not sure if/how this is possible at the moment.  
I tried to do it myself using using https://www.ag-grid.com/javascript-grid-row-height, but was not lucky.

Maybe it's possible to create a setRowHeight method on the ag-grid, similar to BBjGrid?
This way we would be able to do the most common scenario ; set the default rowheight once when initializing the grid.

**Comment by hyyan** [comment]
@thomrob row height feature is not available for BBj yet, but the feature makes sense and we will implement it soon.

**Comment by hyyan** [comment]
@thomrob this is implemented in dev now, please see `Demo.bbj` to learn the usage 

**Methods**
1. **setRowHeight(height!)** : Set the row height globally
2. **setRowHeight(row! , height!) :** set the height for a specific row (works only after `setData`)

**Comment by thomrob** [comment]
thank you! The first method is working, but the second one looks like it's not working here. (although not a problem for me as i don't need to set individual rowheights).
I''m trying to pick development again next months, happy to use the ag-grid as a replacement for some slow/large bui-grids.

**Comment by hyyan** [comment]
@thomrob `Demo.bbj` uses the `setRowHeight(row! , height!)` to change the first-row height when you select the `balham-dark` theme. [Please check that](https://github.com/BBj-Plugins/BBjGridExWidget/blob/dev/Demo/Demo.bbj#L163) and feel free to open another issue if this still does not work for.

**Comment by thomrob** [comment]
thanks, i does work in the demo.bbj.
